src/python/zquantum/optimizers/scipy_optimizer.py

```diff
@@ -47,4 +47,6 @@ def default_callback(params):
         result.history = history
         if 'hess_inv' in result.keys():
             del result['hess_inv']
+        if 'final_simplex' in result.keys():
+            del result['final_simplex']
```

Comment:
How come this is coming about now and wasn't a problem in the past?

Reply:
No idea, I think no one used Nelder-Mead for quite some time :)